service/src/main/java/com/epam/esm/service/entitydtomapper/impl/UserMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.epam.esm.service.entitydtomapper.impl;
+
+import com.epam.esm.dto.UserDTO;
+import com.epam.esm.entity.User;
+import com.epam.esm.service.entitydtomapper.DtoMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper implements DtoMapper<User, UserDTO> {
+
+    @Override
+    public User changeToEntity(UserDTO user) {
+        User u = new User();
```

Comment:
Just FYI: http://modelmapper.org/getting-started/

Reply:
thanks for ModelMapper) I will implement it in 4 module